supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc

```diff
@@ -80,6 +80,17 @@ JBoss, or something else.
 
 *See also*: xref:action[action], xref:identity[identity]
 
+[discrete]
+[[backing store]]
```

Comment:
Can't have spaces in the anchor ID (the rendered build isn't showing properly), so it should be `[[backing-store]]`

This is the case for all of the multi-word entries that were added. Can you please review them all and add the dashes as needed?